mkdocs_monorepo_plugin/plugin.py

```diff
@@ -28,6 +28,7 @@ def __init__(self):
     def on_config(self, config):
         # If no 'nav' defined, we don't need to run.
         if not config.get('nav'):
+            self.originalDocsDir = config['docs_dir']
```

Comment:
I'm interested in knowing more about the use case for this. The way I understand it is that the `!import` statements needs to be configured in the 'nav'. So if you do not have a nav, its almost like we don't want to do anything and bubble some feedback up to the user that a nav is required to use this plugin?

Reply:
We use Backstage, which makes our `mkdocs` depend on [`mkdocs-tech-core`](https://github.com/backstage/mkdocs-techdocs-core).

Some pages were added to our documentation without properly editing the navigation. After reading the [documentation of `mkdocs`](https://www.mkdocs.org/user-guide/configuration/#nav), I discovered that if you omit `nav`, (quoting the doc) it will contain an alphanumerically sorted, nested list of all the Markdown files found within the `docs_dir` and its sub-directories, which is ideal for us.

Upon removing `nav`, running `mkdocs serve` failed, which made me submit this fix 😄

Reply:
@gferon Understand, it makes sense to me! Thanks for giving a bit more context. If you can add a test that would be great!

Reply:
Hi @gferon, just for follow-up, want to add tests? Thanks in advance 🙂

Reply:
Thanks for reminding me! I had a quick look, and I'm actually not sure how I could easily add a `bats` test because the bug only affects `mkdocs serve` and per my understanding, most tests verify that docs can be built using `mkdocs build`.

Reply:
Do you think we can implement some unittests? just to test the on_confg? I found some examples for the search plugin https://github.com/mkdocs/mkdocs/blob/master/mkdocs/tests/search_tests.py#L208.